Library/Formula/awscli.rb

```diff
@@ -96,6 +101,9 @@ def install
   end
 
   def caveats; <<-EOS.undent
+    Python >2.7.2 is recommended to prevent urllib3 SSL warnings:
```

Comment:
This is only relevant on 10.7, which ships 2.7.1. We could upgrade the conditional on `depends_on :python` to `<= :lion`.